skynet/modules/ttt/summaries/app.py

```diff
@@ -16,6 +19,26 @@
 app = create_app()
 app.include_router(v1_router)
 
+post_requests_counter = 0
+
+if echo_requests_base_url:
+
+    @app.middleware("http")
+    async def echo_requests(request: Request, call_next):
+        if request.method == 'POST':
+            global post_requests_counter
```

Comment:
Rather than a globakl counter, we can use a random number and roll the dice on each request.